### DISCUSSION TITLE
An official Discord server for CustomRP

### DISCUSSION THREAD

**Discussion by josephisticated** [maximmax42/Discord-CustomRP#100] [Ideas]
I know it might a bit hard to moderate or time consuming for some people but it'd be really nice if CustomRP had an official Discord server, that way people could exchange ideas and share their custom rich presence with each other.

**Comment by maximmax42** [comment]
Well, there is a server for translators (https://www.customrp.xyz/discordserver, please join to get notified of new terms), but as for a server for everyone - I'm not sure it's that necessary. Both bug reports and ideas can be sent here on GitHub, but then people who use the app are guaranteed to have a Discord account but not necessarily a GitHub one. Maybe in the future, I don't know.
